azureiai/managed_apps/confs/variant/feature_availability.py

```diff
@@ -60,9 +63,14 @@ def set(
                 "priceSchedules": [
                     {
                         "isBaseSchedule": False,
-                        "marketCodes": ["US"],
+                        "marketCodes": ["CA", "US"],
                         "friendlyName": "free_priceOverrideSchedule_US",
                         "schedules": [
+                            {
+                                "retailPrice": {"openPrice": 0, "currencyCode": "USD"},
```

Comment:
Would this support subscriptions with currency set to CAD (Canadian dollars)? My subscription is set to use CAD.

Reply:
I can test with USD to start with.